### DISCUSSION TITLE
Feature request: Use regex for ingredient import automation

### DISCUSSION THREAD

**Discussion by CrackSlowth** [TandoorRecipes/recipes#2574] [Ideas]
As discussed with @smilerz in Discord:
Is it possible to remove some custom keywords from imported recipes?
For instance, I have a lot of recipes that contains "2 tbsp of garlic", "2kg of beef" (in french btw)
Resulting in ingredients called "of garlic" and "of beef"

Regex could be used for ingredients instead of plain replacement
To avoid breaking any existing automation, unused param 3 could be checked and if it equals to "regex" then it does the re.sub, otherwise normal/old replacement

**Comment by vabene1111** [comment]
i created an issue from this, definitely an enhancement for the future.
